tarsier/tag_utils.ts

```diff
@@ -386,3 +388,36 @@ const showMapElements = () => {
     (element as HTMLElement).style.opacity = element.getAttribute('data-original-opacity') || '1';
   });
 }
+
+window.hideElements = () => {
```

Comment:
Might make sense to update naming here

```suggestion
window.hideNonTagElements = () => {
```